src/include/main/database.h

```diff
@@ -10,6 +10,8 @@
 namespace kuzu {
 namespace main {
 
+enum class AccessMode : uint8_t { UNDEFINED = 0, AUTOMATIC = 1, READ_ONLY = 2, READ_WRITE = 3};
```

Comment:
What is `AUTOMATIC` mode? It doesn't seem to be handled. Can we get rid of it for now?